app/Http/Controllers/Internlink/InternlinkController.php

```diff
@@ -110,7 +110,7 @@ public function saveInternship(Request $request)
     {
         $form = $this->form(InternshipForm::class);
         $form->validate(Internship::rules(), [
-            'dateformat' => 'Date should be a valid date of the format YYYY-MM',
+            'dateformat' => 'Date should be a valid date of the format YYYY-MM-DD',
```

Comment:
Yup it's odd! On the bright side, it's definitely not a unique problem (: went to google and apparently people get around this by storing incomplete dates (e.g. 2016-10-00) - see http://stackoverflow.com/questions/9134497/mysql-datatype-to-store-month-and-year-only

I'm not sure how this will work out on the form bit of things though. If the text field doesn't work, one alternative that could work is having a dropdown for year and month (although that doesn't seem like the best solution available)...